packages/ocap-kernel/src/remotes/network.ts

```diff
@@ -1,243 +1,142 @@
-      webSockets(),
-      webTransport(),
-      webRTC({
-        rtcConfiguration: {
-          iceServers: [
-            {
-              urls: [
-                'stun:stun.l.google.com:19302',
-                'stun:global.stun.twilio.com:3478',
-              ],
-            },
-          ],
-        },
-      }),
-      circuitRelayTransport(),
-    ],
-    connectionEncrypters: [noise()],
-    streamMuxers: [yamux()],
-    connectionGater: {
-      // Allow private addresses for local testing
-      denyDialMultiaddr: async () => false,
-    },
-    peerDiscovery: [
-      bootstrap({
-        list: knownRelays,
-      }),
-    ],
-    services: {
-      identify: identify(),
-      ping: ping(),
-    },
-  });
-
-  // Detailed logging for libp2p events. Uncomment as needed. Arguably this
-  // should be controlled by an environment variable or some similar kind of
-  // runtime flag, but probably not worth the effort since when you're debugging
-  // you're likely going to be tweaking with the code a lot anyway.
-  /*
-  const eventTypes = [
-    'certificate:provision',
-    'certificate:renew',
-    'connection:close',
-    'connection:open',
-    'connection:prune',
-    'peer:connect',
-    'peer:disconnect',
-    'peer:discovery',
-    'peer:identify',
-    'peer:reconnect-failure',
-    'peer:update',
-    'self:peer:update',
-    'start',
-    'stop',
-    'transport:close',
-    'transport:listening',
-  ];
-  for (const et of eventTypes) {
-    libp2p.addEventListener(et as keyof Libp2pEvents, (event) => {
-      if (et === 'connection:open' || et === 'connection:close') {
-        const legible = (raw: any): string => JSON.stringify({
-          direction: raw.direction,
-          encryption: raw.encryption,
-          id: raw.id,
-          remoteAddr: raw.remoteAddr.toString(),
-          remotePeer: raw.remotePeer.toString(),
-        });
-        logger.log(`@@@@ libp2p ${et} ${legible(event.detail)}`, event.detail);
-      } else if (et === 'peer:identify') {
-        const legible = (raw: any): string => JSON.stringify({
-          peerId: raw.peerId ? raw.peerId.toString() : 'undefined',
-          protocolVersion: raw.protocolVersion,
-          agentVersion: raw.agentVersion,
-          observedAddr: raw.observedAddr ? raw.observedAddr.toString() : 'undefined',
-          listenAddrs: raw.listenAddrs.map((addr: object) => addr ? addr.toString() : 'undefined'),
-          protocols: raw.protocols,
-        });
-        logger.log(`@@@@ libp2p ${et} ${legible(event.detail)}`, event.detail);
-      } else if (et === 'transport:listening') {
-        const legible = (raw: any): string => JSON.stringify(raw.getAddrs());
-        logger.log(`@@@@ libp2p ${et} ${legible(event.detail)}`, event.detail);
-      } else {
-        logger.log(`@@@@ libp2p ${et} ${JSON.stringify(event.detail)}`, event.detail);
-      }
-    });
-  }
-  */
+  // Initialize the connection factory
+  await connectionFactory.initialize();
```

Comment:
I'm thinking perhaps `ConnectionFactory` should follow the pattern (as found in, e.g., `Kernel`, `VatHandle`, and `RemoteHandle`) of making the constructor and initialization methods private and providing a public static `make` method so that the uninitialized object is never exposed publicly.